docs/rancher/virtualization-management.md

```diff
@@ -6,62 +6,62 @@ keywords:
+- **[Cluster and Project Roles]**(https://rancher.com/docs/rancher/v2.6/en/admin-settings/rbac/cluster-project-roles/):
+    - Define user authorization inside the specific cluster or project where users are assigned the role.
 
 Both global permissions and cluster and project roles are implemented on top of [Kubernetes RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/). Therefore, enforcement of permissions and roles is performed by Kubernetes.
 
-- A cluster owner have full control over the cluster and all resources inside it, e.g. like hosts, VMs, volumes, image, networks, backups, and settings
-- A project user can be assigned to a specific project with permission to manage the resources inside the project
+- A cluster owner has full control over the cluster and all resources inside it, e.g., hosts, VMs, volumes, images, networks, backups, and settings.
+- A project user can be assigned to a specific project with permission to manage the resources inside the project.
 
 
-### Example
+### Multi-Tenancy Example
 The following example provides a good explanation of how the multi-tenant feature works:
 
-1. Firstly, add new users via the Rancher `Users & Authentication` page, click `Create` to add two new separated users e.g. the `project-owner` and `project-readonly` respectively.
-    - A `project-owner` is a user with permission to manage a list of resources of a particular project e.g, the default project.
-    - A `project-readonly` is a user with read-only permission of a particular project e.g, the default project.
+1. First, add new users via the Rancher `Users & Authentication` page. Then click `Create` to add two new separated users, such as `project-owner` and `project-readonly` respectively.
+    - A `project-owner` is a user with permission to manage a list of resources of a particular project, e.g., the default project.
+    - A `project-readonly` is a user with read-only permission of a particular project, e.g., the default project.
     ![](./assets/create-user.png)
-2. Click one of the imported Harvester cluster and visiting the Harvester UI
-    - click the `Projects/Namespaces` tab
-    - select a project e.g, `default` and click the `Edit Config` menu to assign the users to this project with appropriate permissions,  e.g. the `project-owner` user will be assigned with project owner role.
+1. Click one of the imported Harvester clusters after navigating to the Harvester UI.
+    - Click the `Projects/Namespaces` tab.
+    - Select a project such as `default` and click the `Edit Config` menu to assign the users to this project with appropriate permissions. For example, the `project-owner` user will be assigned the project owner role.
    ![](./assets/add-member.png)
-4. Continue to add the project-readonly user to the same project with read-only permissions and click save.
+1. Click to **Add** the `project-readonly` user to the same project with read-only permissions and click **Save**.
```

Comment:
Isn't "continue to add" in this context referring to repeating the step in line 56 and not so much `Add` being an actual element that can be clicked. If it is an element users can click, then it should be mentioned on line 56 too? in which case this should be `Click **Add**` without the `to`)